nightfall-client/docker-entrypoint.sh

```diff
@@ -3,7 +3,7 @@ mongod --dbpath /app/mongodb/ --fork --logpath /var/log/mongodb/mongod.log --bin
 while ! nc -z localhost 27017; do sleep 3; done
 echo 'mongodb started'
 
-if [ -z "${USE_INFURA}" ]
+if [ -z "${USE_INFURA}" ] && [ -z "${USE_HOSTED_GETH}" ];
```

Comment:
I might be wrong. But should this be `||` instead of `&&`?

Reply:
-z check for env variable `if doesn't exist or set`